packages/api/tests/endpoints/collections/create-collection.js

```diff
@@ -110,10 +110,7 @@ test.beforeEach(async (t) => {
     Endpoint: QueueArn,
   }).promise();
 
-  await sns().confirmSubscription({
-    TopicArn,
-    Token: SubscriptionArn,
-  }).promise();
```

Comment:
When invalid token is provided, we get error: message: 'Invalid parameter: Token'.
To get correct token, we need to get the notification message of the subscription from the subscribed SQS queue, however, there isn't such SQS message in localstack env.  So this confirmSubscription is deleted.

Reply:
what are the ramifications of not verifying that this functionality is working in these tests? I'm made a little uncomfortable by lack of parity between testing and operational environment here, but I presume you've done the research to be sure this can't/shouldn't be done as it was before? I assume it was present in older localstack

Reply:
The check was added in https://github.com/nasa/cumulus/pull/2432, and yeah, the arn is not a token. 

I think it's important to understand what confirmSubscription does - it verifies the endpoint owner's intent to get messages by forcing them to hit the API to confirm.     You don't get a token *back* for sqs, because you're the owner, IAM perms, etc.      You'd expect to need this for something like an 'email' subscription.  

I'm thinking the use of this pattern is a mistake, assuming I'm reading the AWS docs correctly.     We should consider removing it elsewhere, particularly where SQS queues are used.